diff -urN echat-0.04beta1.org/curs_strn.c echat-0.04beta1/curs_strn.c
--- echat-0.04beta1.org/curs_strn.c	2003-08-20 15:11:25.000000000 +0200
+++ echat-0.04beta1/curs_strn.c	2003-08-20 15:11:53.000000000 +0200
@@ -267,6 +267,7 @@
         case KEY_DC:
             string_del(); break;
         case KEY_BACKSPACE:
+	case 127:
             string_bs(); break;
         case ENTER:
             screen_bot();
